packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_view.dart

```diff
@@ -99,10 +102,13 @@ class _PropertiesListState extends State<_PropertiesList> {
   Widget build(BuildContext context) {
     return Column(
       children: <Widget>[
+        _FilterControls(controller: widget.controller),
+        if (widget.editableProperties.isEmpty)
```

Comment:
what if editableProperties is empty and the filter query is empty? In this case, this message may not make sense

Reply:
That is actually handled before we get here - if there are no editable properties at all we display a message instead of rendering the `_PropertiesList`: https://github.com/flutter/devtools/blob/d46c813c4f9cef2cb71ab8a5c875d41c63526a35/packages/devtools_app/lib/src/standalone_ui/ide_shared/property_editor/property_editor_view.dart#L58